Source/Schema.NET/OneOrMany{T}.cs

```diff
@@ -238,6 +238,38 @@ public IEnumerator<T> GetEnumerator()
         /// <returns>An enumerator for the <see cref="OneOrMany{T}"/>.</returns>
         IEnumerator IEnumerable.GetEnumerator() => this.GetEnumerator();
 
+        /// <summary>
+        /// Creates an array from <see cref="OneOrMany{T}" />.
+        /// </summary>
+        /// <returns>An array containing all the elements.</returns>
+        public T[] ToArray()
+        {
+            if (this.HasOne)
+            {
+                return new[] { this.collection[0] };
+            }
+            else if (this.HasMany)
+            {
+                var result = new T[this.collection.Length];
+                Array.Copy(this.collection, 0, result, 0, this.collection.Length);
+                return result;
+            }
+            else
+            {
+#if NETSTANDARD1_1
+                return new T[0];
+#else
+                return Array.Empty<T>();
+#endif
+            }
+        }
+
+        /// <summary>
+        /// Returns the first item from <see cref="OneOrMany{T}"/> or default <typeparamref name="T"/>.
+        /// </summary>
+        /// <returns>The first element in the <see cref="OneOrMany{T}"/></returns>
+        public T FirstOrDefault() => this.Count > 0 ? this.collection[0] : default;

```

Comment:
We'll get this from `System.Linq` because we implement `IEnumerable<T>`. Do you still think it's worth implementing?

Reply:
Yeah - I really only implemented it to try and satisfy the linting rule about having methods for implicit actions. It would be faster than [the default `FirstOrDefault`](https://github.com/dotnet/corefx/blob/master/src/System.Linq/src/System/Linq/First.cs#L43-L77) implementation but at the same time, probably not actually worth the effort of keeping if we don't aim for the other linting rules. I'll remove it in another commit soon.

Reply:
Cool. I'm happy for this to be squash merged after this change. Feel free to do so.